### PR TITLE
Add Titouan as an editor.

### PR DESCRIPTION
**Pull request by mikewest**
@letitz WDYT?

### CODE REVIEW

**Comment by mikewest** [comment]
(I didn't regenerate the HTML, as I assume you'll be doing that soon for another PR, and I don't want to cause conflicts.)

File: index.src.html [hunk 1]
```diff
@@ -5,7 +5,8 @@ <h1>CORS and RFC1918</h1>
 ED: https://wicg.github.io/cors-rfc1918/
 Shortname: CORS-RFC1918
 Level: 1
-Editor: Mike West 56384, Google Inc., mkwst@google.com
+Editor: Titouan Rigoudy, Google, titouan@google.com
```
Comment (letitz) [inline index.src.html:8]: Is 56384 your W3C member ID? I can find mine if that would be helpful.

**Review by letitz — APPROVED** [review]
I think I would be honored, thanks for sending a patch Mike!

File: index.src.html (hunk 1 above)
Comment (mikewest) [reply index.src.html:8]: Yes. There's some metadata linking behind the scenes somewhere. If you have an ID, I'd be happy to put it in. If it's at all annoying to find, add it later?

Comment (letitz) [reply index.src.html:8]: Not at all annoying, here it is: 124486.

**Pull request merged**